### PR TITLE
Revamp onboarding section

### PR DESCRIPTION
**Pull request by gionn**
- [X] Split onboarding page in multiple pages
- [X] Update plans section according to the current frontend

### CODE REVIEW

File: docs/onboarding-extra-resources.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+sidebar_label: Extra-resources
+---
+
+In this section you will learn what extra resources are and how you can add them
+to your product plans.
+
+## What is an Extra Resource?
+
+Extra Resources refer to _goods_ or _services_ which can be sold together with
+the application (for example, a 10-days pack of Technical Support, hardware
```
Comment (pimuzzo) [inline docs/onboarding-extra-resources.md]:
```suggestion
the application (for example, a 10-days pack of technical support, hardware
```

File: docs/onboarding-extra-resources.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+### Tiered scheme
+
+Tiered scheme means that every unit charge is calculated with its own tier
+price.
+
+With tiered pricing, once you fill up a *tier* you move to the next tier and
+start charging a different price.
+
+Tiered pricing is different from volume pricing because tiered pricing defines a
+price per unit withing a range, while volume pricing defines a price for all
```
Comment (pimuzzo) [inline docs/onboarding-extra-resources.md]:
```suggestion
price per unit within a range, while volume pricing defines a price for all
```

File: docs/billing.md [hunk 3]
```diff
@@ -229,15 +134,15 @@ Furthermore, Cloudesire supports an additional type of coupon:
...
-etc.). On top the platform allows to create several **plans** for the same
+etc.). On top the platform allows to create several _plans_ for the same
 bundle (e.g. _silver_, _gold_, _platinum_, etc.).
 
-Each _bundle plan_ is defined as a composition of _products plans. _Using a
+Each _bundle plan_ is defined as a composition of _product plans_. Using a
 simple interface it's possible to specify for each of them a **discount
 percentage**; in this way, the total price of a _bundle plan_ is calculated as
-the sum of all the discounted prices of all the _products plans attached to
+the sum of all the discounted prices of all the _products plans_ attached to
```
Comment (pimuzzo) [inline docs/billing.md]:
```suggestion
the sum of all the discounted prices of all the _product plans_ attached to
```

File: docs/onboarding-extra-resources.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+The next step is to define the quantity ranges and their pricing. Go to the
+"_Extra Resources Values_" section and fill:
+
+* _Resource Name_: select from the combo-box on the left a previously created
+  Extra Resource
+* _Min_: Start quantity for the current interval (should not overlap the
+  previous one)
+* _Max_: End quantity for the current interval (may be empty in the last range to avoid defining an upper bound)
+* _Step_: should be greater than 1. Decide which are the effective quantities
```
Comment (pimuzzo) [inline docs/onboarding-extra-resources.md]:
```suggestion
* _Step_: should be greater than or equal to 1. Decide which are the effective quantities
```

File: docs/onboarding-plans.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+* _Coupon configuration_: whether using coupons is globally available for this
+  plan or not. An option to require a coupon in order to buy a plan is also
+  available
+* _Upgrades available_: wheter a subscription of this plan can be upgraded to
+  another one (e.g. upgrades are not feasible or are without meaning)
+
+### Extra-Resources
+
+_Extra-resources_ are goods or services which can be sold together with the
+product. Examples are a 10-days pack of Technical Support, some hardware
```
Comment (pimuzzo) [inline docs/onboarding-plans.md]:
```suggestion
product. Examples are a 10-days pack of technical support, some hardware
```

File: docs/pricing-models.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+sidebar_label: Pricing Models
+---
+
+In this section we will focus on pricing models. The Pricing Model configuration
+section is one of the most important aspects of product configuration. Since it
+directly influences your product sales, it is important to understand which the
+most common pricing models available on the platform.
+
+You can deeply customize the pricing model for your product in Cloudesire (for
+example, you can add extra resources to your product LINK), but the choice of
```
Comment (pimuzzo) [inline docs/pricing-models.md]: missing link! :fire:

File: docs/pricing-models.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+the right pricing model is not always easy. That's why in the following
+paragraphs you will find an introduction to the most used pricing models in SaaS
+businesses.
+
+## Available pricing models
+
+There are many different basic pricing models that you can implement in
+Cloudesire. To define a pricing model, you need at least one product into your
+catalog. Select the product, then "_Edit_" and go to the "_Plans_" tab.You can
```
Comment (pimuzzo) [inline docs/pricing-models.md]:
```suggestion
catalog. Select the product, then "_Edit_" and go to the "_Plans_" tab. You can
```

File: docs/billing.md [hunk 8]
```diff
@@ -278,9 +183,14 @@ by the platform and vendors should not worry about it.
 ### Backup
 
 * Cloudesire allows the customers to buy **backup plans.**
-* every backup plan includes to a maximum number of backups executions (configurable in the platform) that can be manually performed by the customers or scheduled
+* every backup plan includes to a maximum number of backups executions
```
Comment (malteo) [inline docs/billing.md]: this is not clear

File: docs/api-product.md [hunk 9]
```diff
@@ -11,7 +11,7 @@ executed.
 An example API product is available on our [demo marketplace](https://demo.cloudesire.com/281/api/api-demo)
 
 The onboarding process starts after creating a new "Product" in the Marketplace
-Catalog, as described [here](onboarding.md#applications-catalog-marketing-sales-onboarding).
+Catalog, as described in the [onboarding](onboarding.md) section.
```
Comment (malteo) [inline docs/api-product.md]:
```suggestion
Catalog, as described in the [onboarding section](onboarding.md).
```

File: docs/glossary.md [hunk 10]
```diff
@@ -44,14 +44,14 @@ A user that buys products in the Cloudesire Marketplace.
 
 An user role can manage catalogs of products resold by resellers.
 
-## Extra resource
+## Extra-Resources
```
Comment (malteo) [inline docs/glossary.md]: no need for the hyphen

```suggestion
## Extra Resources
```

File: docs/glossary.md [hunk 11]
```diff
@@ -44,14 +44,14 @@ A user that buys products in the Cloudesire Marketplace.
...
 
 A configurable, usually optional characteristic of a product that has its own
 configurable pricing (e.g. a 10-days pack of Technical Support, some hardware
 components, etc), that can be prepaid or postpaid if tied to an application
 metric.
 
-Go to [extra resource section](onboarding.md#extra-resources)
+Go to [Extra-Resources](onboarding-extra-resources.md) section.
```
Comment (malteo) [inline docs/glossary.md]:
```suggestion
Go to [Extra Resources section](onboarding-extra-resources.md).
```

File: website/i18n/en.json [hunk 12]
```diff
@@ -89,6 +97,10 @@
         "title": "Platform modules",
         "sidebar_label": "Platform modules"
       },
+      "pricing-models": {
+        "title": "SaaS pricing models explained",
+        "sidebar_label": "Pricing Models"
+      },
```
Comment (malteo) [inline website/i18n/en.json:103]: I still think there's no need for the hyphen in "Extra resources"

plus be consistent with the capital letters

**Review by malteo — CHANGES_REQUESTED** [review]

**Review by manuelmazzuola — APPROVED** [review]

**Review by malteo — APPROVED** [review]

**Pull request merged**